config/crd/bases/infrastructure.cluster.x-k8s.io_domachines.yaml

```diff
@@ -182,6 +182,18 @@ spec:
               ready:
                 description: Ready is true when the provider resource is ready.
                 type: boolean
+              volumes:
+                description: Volumes contains the DigitalOcean droplet associated
+                  block storage volumes.
+                items:
+                  description: DOVolume defnies a DO Block Storage Volume.
```

Comment:
typo: defnies -> defines

Reply:
Looks like I forgot to re-render the yamls after the fix. Thx.